docs/docsite/rst/dev_guide/testing/sanity/shebang.rst

```diff
@@ -4,13 +4,12 @@ shebang
 Most executable files should only use one of the following shebangs:
 
 - ``#!/bin/sh``
-- ``#!/bin/bash``
+- ``#!/bin/bash -eu``
+- ``#!/bin/bash -eux``
 - ``#!/usr/bin/make``
 - ``#!/usr/bin/env python``
 - ``#!/usr/bin/env bash``
 
-NOTE: For ``#!/bin/bash``, any of the options ``eux`` may also be used, such as ``#!/bin/bash -eux``.
-
 This does not apply to Ansible modules, which should not be executable and must always use ``#!/usr/bin/python``.
```

Comment:
Is this still accurate? As far as I know, Python ansible modules no longer need `#!/usr/bin/python` and even core has gone ahead and remove all shebangs from its modules.

Reply:
It's not, but I was trying to limit my changes to those needed to address the originally reported issue -- since there are quite a few other things out-of-date in this doc.